collections/cnes-land-cover-map.yaml

```diff
@@ -0,0 +1,117 @@
+Name: CNES Land Cover Map
+Description: |
+  The CNES Land Cover Map (Occupation des Sols, OSO) produces land classification for Metropolitan France at 10 m spatial resolution based on Sentinel-2 L2A data within the Theia Land Cover CES framework. 
+  Maps for 2020, 2019, and 2018 use a 23-categories nomenclature. For earlier maps in 2017 and 2016, a fully compatible 17-classes nomenclature is employed.
+AdditionalInfoExternal:
+    Title: Additional info
+    Path: cnes-land-cover-map/README.MD 
+Image: cnes-land-cover-map/cnes-land-cover-map-france-overview.png
+#EDCBrowser: https://sentinelshare.page.link/kP8Y
```

Comment:
I think this should be
- `EOBrowser`
- only be added once the collection is available in EO Browser